rundeckapp/grails-spa/packages/ui-trellis/src/app/components/home/HomeActionsMenu.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <dropdown menu-right append-to-body ref="dropdown">
+    <a role="button" class="as-block link-hover link-quiet link-block-padded text-inverse dropdown-toggle">
+      {{ $t('button.Action') }}
+      <span class="caret"></span>
+    </a>
+    <template #dropdown>
+      <template v-for="(option, index) in availableOptions">
```

Comment:
Maybe an improvement to add: if the user has no admin or job create permissions, this drop down is blank (doesn't show anything when clicked on).  This matches the old behavior, but is maybe not the best behavior... Probably it should not be shown in that case